website/src/data/users.js

```diff
@@ -269,4 +269,16 @@ const users = [
   },
 ];
 
+users.forEach((user) => {
+  if (
+    !user.preview ||
+    user.preview.startsWith('http') ||
+    user.preview.startsWith('//')
```

Comment:
CI fails because of this condition:

```
4:20:00 PM: TypeError: user.preview.startsWith is not a function
4:20:00 PM:     at Array.forEach (<anonymous>)
4:20:00 PM: (undefined) TypeError: user.preview.startsWith is not a function
4:20:00 PM:     at module.exports.users.forEach.user (main:35114:6781)
4:20:00 PM:     at Array.forEach (<anonymous>)
4:20:00 PM:     at Module.<anonymous> (main:35114:6735)
4:20:00 PM:     at __webpack_require__ (main:21:30)
4:20:00 PM:     at Promise.resolve.then (main:13011:7767)
```